File: python/ex25.py
a = float(input())
b = float(input())
c = float(input())

if c>a+b or a>b+c or b>c+a:
    print("INVALIDO")
else: 
    if a!=b and b!=c and a!=c:
        print("ESCALENO")
    elif a==b==c:
        print("EQUILATERO")
    elif a==b or b==c or c==a:
        print ("ISOSCELES")
     

File: c/ex25.c
//Lê o tamanho dos lados de um triângulo e classifica-o
#include<stdio.h>
int main(){
    int a, b, c;
    scanf(" %i %i %i", &a, &b, &c);
    if((a + b) < c || (a + c) < b || (b + c) < a){
        printf("INVALIDO");
    }
    else if(a == b && b == c){
        printf("EQUILATERO");
    }
    else if(a == b || b == c || c == a){
        printf("ISOSCELES");
    }else{
        printf("ESCALENO");
    }
    return 0;
}